client/src/app/app.component.ts

```diff
@@ -40,35 +42,20 @@ export class AppComponent implements OnInit {
   }
 
   ngOnInit() {
-    this.activatedRoute.queryParams.subscribe(params => {
```

Comment:
``this.activatedRoute`` can also be removed from the constructor

Reply:
Done